sct.py

```diff
@@ -932,7 +939,31 @@ def collect_logs(test_id=None, logdir=None, backend=None, config_file=None):
     click.echo(table.get_string(title="Collected logs by test-id: {}".format(collector.test_id)))
 
 
+# pylint: disable=too-many-locals,too-many-statements,too-many-arguments
+def send_perf_email(test_id, email_recipients, test_results, testrun_dir, start_time, es_doc_type='test_stats'):
+    reporter = BaseResultsAnalyzer(es_index=test_id, es_doc_type=es_doc_type, email_recipients=email_recipients)
+    for subject, content in test_results.items():
+        if 'email_body' not in content:
+            content['email_body'] = dict()
+        if 'attachments' not in content:
+            content['attachments'] = []
+        if 'template' not in content:
+            content['template'] = None
+        email_content = content.copy()
```

Comment:
maybe it is better to use deepcopy()? because copy() doesn't copy nested list/dicts starting from level 2

Reply:
done